libvirt/tests/src/virtual_network/qemu/pktgen_burst_mode_test.py

```
@@ -0,0 +1,147 @@
+            params, vm_image_path=vm_image_paths, iface_list=iface_list)
+
+        return vm_list
+
+    def run_test(vm_list):
+        """
+        Run pktgen in burst mode for vm.
+        """
+        result_file = write_in_result_file(test, params)
+
+        if disable_iptables_rules_cmd:
+            process.system(disable_iptables_rules_cmd, shell=True)
```

Ruff (0.13.1)

Function call with `shell=True` parameter identified, security issue

(S604)